src/engine/sys/sdl_glimp.cpp

```diff
@@ -1758,6 +1759,14 @@ static void GLimp_SetAppMetadataProperty( const char* name, const char* value )
 	}
 }
 
+static void GLimp_SetHint( const char* name, const char* value )
+{
+	if ( !SDL_SetHint( name, value ) )
+	{
+		logger.Warn( "SDL_SetHint( %s, %s ) failed: %s\n", name, value, SDL_GetError() );
```

Comment:
Log message shouldn't have `\n`

Is it really possible for this to fail anyway? In the SDL wiki's example they don't check it. https://wiki.libsdl.org/SDL3/SDL_SetHint